src/displayapp/screens/settings/QuickSettings.cpp

```diff
@@ -10,8 +10,12 @@ using namespace Pinetime::Applications::Screens;
 namespace {
   void ButtonEventHandler(lv_obj_t* obj, lv_event_t event) {
     auto* screen = static_cast<QuickSettings*>(obj->user_data);
-    if (event == LV_EVENT_CLICKED) {
-      screen->OnButtonEvent(obj);
+    static lv_obj_t* pressedObj = nullptr;
```

Comment:
I have a bad feeling with introducing a local static variable, but that might just be me. Is there any other way to implement this? I am just asking and would like your opinion.

Reply:
I think it's not a problem. The compiler should simply allocate some space near the function itself. I am not 100% sure tough. We should ask some experienced programmer. 
Regarding the implementation, there surely are other ways to implement this. I can think of a global variable (although scoped) but I think it's just worse, otherwise it's possible to add another member to the class, but it would be unnecessary, as I see this.
In conclusion, I guess a static variable is the simplest and safest way of doing this, tbh

Reply:
Ok, I agree with you.